tests/core/evm/vm/test_message.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import unicode_literals
+import pytest
+import evm.vm.message as m
+
+
+def _create_message(gas=1, gas_price=1, to=b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff", sender=b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff", value=1, data=b"", code=b"", origin="\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff"):
+    return m.Message(gas=gas, gas_price=gas_price, to=to, sender=sender, value=value, data=data, code=code, origin=origin)
```

Comment:
For readability, this codebase is using the following style when calling or instantiating things with many arguments.  Generally, one per line.

```python
m.Message(
    gas=gas,
    gas_price=gas_price,
    to=to,
    ....
)
```